templates/includes/sidebar.html

```diff
@@ -5,9 +5,9 @@
   {% else %}
     <li class="list-group-item"><a href="{% url 'login'%}?next={{request.path}}">Login</a></li>
```

Comment:
Ensure that the `login` URL pattern is correctly defined in your project's `urls.py` or included from Django's authentication URLs.